pkg/types/service.go

```diff
@@ -271,6 +274,16 @@ func convertEnvVars(vars map[string]string) []v1.EnvVar {
 	return envVars
 }
 
+func convertLocalObjects(secrets []string) []v1.LocalObjectReference {
```

Comment:
I think this function could have a "more specific" name, like `setImagePullSecrets`, as it only will be used for the secrets...